### DISCUSSION TITLE
Tab stop support for RTF

### DISCUSSION THREAD

**Discussion by rmcdonald** [empira/PDFsharp#160] [Q&A]
I'm working on applications using `Windows Presentation Foundation`.  I'm using `System.Windows.Controls.RichTextBox` to display RTF generated by `MigraDoc` by calling `RtfDocumentRenderer.RenderToString()` and then calling `RichTextBox.SelectAll()` and `RichTextBox.Selection.Load()`.  The displayed RTF document looks as I expect, except for the absence of tab stops.  In contrast, the PDF file I generate from the same `MigraDoc.DocumentObjectModel.Document` looks as I expect, including tab stops.

I'm declaring tab stops as follows:

            style = docReport.Styles.AddStyle("TabbedTwoColumnPairs", "Normal");
            style.ParagraphFormat.TabStops.Clear();
            style.ParagraphFormat.TabStops.AddTabStop("4cm");
            style.ParagraphFormat.TabStops.AddTabStop("8cm");
            style.ParagraphFormat.TabStops.AddTabStop("12cm");

Searching the web, I found that there is also `System.Windows.Forms.RichTextBox`, but that is apparently only for applications using Windows Forms.  That class has a `SelectionTabs` property that allows specification of tab stops.  `System.Windows.Controls.RichTextBox` does not support this property.

Is there another way to specify tab stops using `MigraDoc`, or does this `MigraDoc` feature depend on using it with a `System.Windows.Forms.RichTextBox`  and some code outside of the development of a `Document` instance?  Running `find . -type f -name "*.cs" | xargs grep -H SelectionTabs` within the `master` branch of the `PDFsharp` repo doesn't show any rows.

**Comment by TH-Soft** [comment]
Tab stops work when watching the RTF with e.g. Microsoft Word.
I guess that RichTextBox has some limitations with respect to tab stops.
